allensdk/brain_observatory/behavior/data_files/stimulus_file.py

```diff
@@ -203,6 +203,38 @@ def session_type(self) -> str:
         """
         return self._retrieve_from_params("stage")
 
+    @property
+    def image_stimulus_name(self) -> str:
```

Comment:
@morriscb  How is this different from https://github.com/AllenInstitute/AllenSDK/blob/rc/2.15.2/allensdk/brain_observatory/behavior/behavior_project_cache/tables/util/metadata_parsers.py#L35 ? It might lead to discrepancies if we're using multiple code paths to grab the same thing ?

Reply:
The returns are slightly different here. The code you link returns the specific image set shown (e.g. image_A). The image_stimulus_name return gives the full name of the image file (e.g. `Natural_Images_Lum_Matched_set_ophys_6_2017`) so they are slightly different in their returns.